### DISCUSSION TITLE
Volume buttons on keyboard does not work

### DISCUSSION THREAD

**Discussion by paviabera** [MonitorControl/MonitorControl#1708] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Nothing happens when using the volume keys on the keyboard or app slider. But the brightness slider works fine. 
Recently, I updated my mac os. Before the update, everything was working fine. 

### Steps to reproduce

1. go to app 
2. use the volume slider (nothing happens)
3.  use the brightness slider (works fine)

### Expected behavior

Volume to respond

### Anything else?

- macOS version: 15.2
- Mac model: 2019 intel i7 macbook pro 16"
- MonitorControl version: 4.3.3
- Monitor(s): BenQ2480 
- Apple Silicon/M1 (yes or no): no

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
Hi there, I am not sure what could be the issue. Does brightness control work between the 50-100% range as well (where DDC control is being used by default)? Is the audio output set to the display?

**Comment by waydabber** [comment]
Also look at the answer here: https://github.com/MonitorControl/MonitorControl/discussions/1710 if your display is in HDR mode.
